src/index.html

```diff
@@ -3,10 +3,40 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link href="https://fonts.googleapis.com/css?family=Faster+One" rel="stylesheet">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>TvShow</title>
 </head>
 <body>
-  <h1>hello</h1>
+  <div class="header">
+    <div class="logo-image">
+      <!-- <img src="/images/Film-Logo-Templates.jpg" class="logo" alt=""> -->
+      <h1 class="logo">Logo</h1>
+    </div>
+    <div class="menu">
+      <ul class="menu-item">
+        <li class="menu-link">Item1</li>
+        <li class="menu-link">Item2</li>
+        <li class="menu-link">Item3</li>
+        <li class="movie-num">Item3</li>
+      </ul>
+    </div>
+
+  </div>
+  <div class="content">
+    <!-- The Data Comes Dynamically  -->
+    <ul class="movie-list"></ul>
+  </div>
+
+  <div class="pupop-comments">
+    <!-- The Data comes Dynamically -->
+  </div>
```

Comment:
Since the comments button is loaded dynamically, I recommend removing this element.